share/functions/__fish_print_packages.fish

```diff
@@ -35,6 +35,17 @@ function __fish_print_packages
         return
     end
 
+    # pkg_info on OpenBSD provides versioning info which we want for
+    # installed packages
+    if begin
```

Comment:
This `begin` and `end` is unnecessary since about 2.3.0. Now `and` and similar bind to `if`.